tests/apple/app_store_connect/test_resource_manager.py

```diff
@@ -18,6 +18,7 @@ class StubFilter(ResourceManager.Filter):
     field_one: Optional[str] = None
     field_two: Optional[StubEnum] = None
     field_three_four: Optional[CustomString] = None
+    field_with__dot__dots: Optional[str] = None
```

Comment:
This isn't relevant any more, is it?

Reply:
yes, removed https://github.com/codemagic-ci-cd/cli-tools/pull/67/commits/1e641150af41c98a49a55bacdfa22af935437d5a#diff-122a8c79fd95bacf0d08d430bcd827e432174b82ddb5ae6746b36287ed453970L21